### DISCUSSION TITLE
Parameter identification Vmax and Km in PK-Sim 11

### DISCUSSION THREAD

**Discussion by Mwilaputa** [Open-Systems-Pharmacology/Forum#1444] [Questions & Problems]
How can I select Vmax or Km for parameter identification in PK-Sim 11? The previous version had the tab "Reaction" on the drop down menu for organs in parameter identification section where you can locate enzyme parameters like Vmax or Km and select for parameter identification. But it seems that PK-Sim 11 does not have the "Reaction" on the menu list. How can I resolve this? N.B. If I try to open a file in version 11 that was saved under version 10, the "reaction" menu appears.

**Comment by StephanSchaller** [comment]
Try using the "search" functionality in the parameter selection tab:
![image](https://user-images.githubusercontent.com/25477643/215548892-0f7262c6-1dfb-40e5-aad3-4b3f00e66f20.png)

**Reply by Mwilaputa** [reply]
Hi Stephan. I have tried using search functionality and manually still I cant find it.

**Reply by StephanSchaller** [reply]
Are you sure this reaction is still part of the model? Maybe you changed the metabolism to a linear model and not a MM kinetic?

**Reply by Mwilaputa** [reply]
I am very sure. The reaction is part of the model and it MM kinetic. Can I maybe share the PK-Sim file so that you check from your side?

**Reply by wilbertdew** [reply]
@Mwilaputa If you select these parameters in the simulation as Favorites, you can then find them back easily by applying the filter "Favorite" in the Parameter Identification or by right clicking the simulation and selecting "Start Parameter Identification", which includes the favorite parameters in the "Identification Parameters" list automatically.
